packages/neuron-wallet/src/services/ckb-runner.ts

```diff
@@ -71,20 +75,40 @@ export const startCkbNode = async () => {
   if (app.isPackaged && process.env.CKB_NODE_ASSUME_VALID_TARGET) {
     options.push('--assume-valid-target', process.env.CKB_NODE_ASSUME_VALID_TARGET)
   }
-  ckb = spawn(ckbBinary(), options, { stdio: ['ignore', 'ignore', 'pipe'] })
+  ckb = spawn(ckbBinary(), options, { stdio: ['ignore', 'pipe', 'pipe'] })
+
   ckb.stderr &&
     ckb.stderr.on('data', data => {
       logger.error('CKB:\trun fail:', data.toString())
       ckb = null
     })
+  ckb.stdout &&
+    ckb.stdout.on('data', data => {
+      if (!app.isPackaged || !process.env.CKB_NODE_ASSUME_VALID_TARGET) {
+        return
+      }
+      const dataString: string = data.toString()
+      if (
+        dataString.includes(
+          `can't find assume valid target temporarily, hash: Byte32(${process.env.CKB_NODE_ASSUME_VALID_TARGET})`
```

Comment:
Better to check `can't find assume valid target temporarily, hash: Byte32(\w{66})` and extract the `assume valid target` from logs. Thus users can use their own CKB nodes with their specified `assume valid target` and get the correct info from Neuron.

Reply:
Is it has an rpc API to get the node logs or get the node running directory?
If someone starts local nodes with `assume-valid-target`, he should know what's means.

Reply:
> Is it has an rpc API to get the node logs or get the node running directory? If someone starts local nodes with `assume-valid-target`, he should know what's means.

My fault, I thought the `ckb` is the running node but it should be undefined when the user uses his own node.